static/app/components/workflowEngine/form.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+export const StickyFooter = styled('div')`
+  position: sticky;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  padding: ${space(2)} ${space(4)};
+  background: ${p => p.theme.background};
+  border-top: 1px solid ${p => p.theme.translucentGray200};
+  box-shadow: 0px -4px 24px 0px rgba(43, 34, 51, 0.12);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Should this use the `Flex` component instead?

Reply:
Trying to avoid an unnecessary wrapper `div`, but you're right, better to stick with established patterns

Reply:
For replicating the wrappers agree, i was even just thinking to pass the flex component into here instead of `div` 😅

Reply:
I gave this a shot and decided against it. It would be a new pattern and unnecessarily complicate the type inference for these component props. Since that's already an issue in our codebase, I'd rather not introduce new patterns that exacerbate things.